lib/dialects/mysql/query-generator.js

```diff
@@ -434,6 +435,24 @@ class MySQLQueryGenerator extends AbstractQueryGenerator {
     return result;
   }
 
+  selectFromTableFragment(options, model, attributes, tables, mainTableAs) {
+    let fragment = `SELECT ${attributes.join(', ')} FROM ${tables}`;
```

Comment:
Just to sanity check, do we need to escape anything here?

Reply:
This is taken directly from abstract/query-generator.js so I don't believe so?